tests/bootstrap_project/test/util.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+import 'package:uuid/uuid.dart';
+
+(String, String) createRandomProjectName(String root) {
```

Comment:
Suggestion: Return as named record or a simple POD class.

The user of this method will need to investigate this and the next method to understand what is being returned where. Provide a name to the returned fields so that this can be inspected from the call site.

Reply:
40d0b0b